packages/react-saasify/src/themes/waves.module.css

```diff
@@ -0,0 +1,709 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:400&display=swap');
```

Comment:
Down the road, we should figure out how to not have assets like these loaded for every scene as part of the default bundle. Definitely not worth worrying about right now, it's just worth thinking about as we consider how to improve our general theming support.